sync_check/main.go

```diff
@@ -35,27 +33,29 @@ func main() {
 	case flag.ErrHelp:
 		os.Exit(0)
 	default:
-		log.Errorf("parse cmd flags err %s\n", err)
+		log.Errorf("parse cmd flags err %s\n", errors.Trace(err))
 		os.Exit(2)
 	}
 
+	log.SetLevelByString(cfg.LogLevel)
+
 	ok := cfg.checkConfig()
 	if !ok {
 		log.Error("there is something wrong with your config, please check it!")
 		return
 	}
 
-	sourceDB, err := util.CreateDB(cfg.SourceDBCfg)
+	sourceDB, err := util.CreateDB(cfg.SourceDBCfg, cfg.SourceSnapshot)
 	if err != nil {
 		log.Fatal(err)
 	}
-	defer util.CloseDB(sourceDB)
+	defer pkgdb.CloseDB(sourceDB)
```

Comment:
why refactor `CloseDB` into `pkgdb`，but `CreateDB` in `util`?

Reply:
done